app/backend/db/crud/crud_utils.py

```diff
@@ -0,0 +1,6 @@
+def put_values_if_not_none(db, obj, **kwargs):
+    for attr, value in kwargs.items():
+        if value is not None:
+            setattr(obj, attr, value)
+    
+    db.commit()
```

Comment:
Add refresh to obj after.
Add new line at the end

Reply:
done